layouts/partials/comments/provider/vssue.html

```diff
@@ -14,7 +14,7 @@
                 props: {
                     title: "{{ $.Title }}",
                     options: {
-                        proxy: (url) => "https://cors-anywhere.azm.workers.dev/" + url,
+                        autoCreateIssue: true,
```

Comment:
I think it might be better to make it as a configurable option

Reply:
Yes, I followed the official default value